### PR TITLE
Issue #33: send JSON to syslog instead of util.inspect output.

### PR DESCRIPTION
**Pull request by FGM**
- Force package.json commands to use Meteor npm instead of system default.

### CODE REVIEW

File: src/Senders/SyslogSender.js [hunk 1]
```diff
@@ -64,8 +72,49 @@ const SyslogSender = class extends SenderBase {
...
+  serializeDefault(doc) {
+    let result;
+    try {
+      result = JSON.stringify(doc);
+    }
+    catch (e1) {
+      const step1 = {
+        level: doc.level,
+        facility: doc.facility,
+        error: `JSON.stringify error: ${e1.message}.`,
```
Comment (yched) [inline src/Senders/SyslogSender.js]: Suggestion : logger_error: 'Cannot serialize logged data:  ${e1.message}' ?

(To make it clearer that what was logged by the application was not an error, but that the logger encountered an error while processing the logged data)

File: src/Senders/SyslogSender.js [hunk 2]
```diff
@@ -64,8 +72,49 @@ const SyslogSender = class extends SenderBase {
...
+   */
+  serializeDefault(doc) {
+    let result;
+    try {
+      result = JSON.stringify(doc);
+    }
+    catch (e1) {
+      const step1 = {
+        level: doc.level,
+        facility: doc.facility,
```
Comment (yched) [inline src/Senders/SyslogSender.js]: We could include doc.message if it is a string ?

**Comment by yched** [comment]
LGTM :-)

**Comment by FGM** [comment]
If you say so :-)

**Pull request closed without merging**